client/app/recordings/new/route.js

```diff
@@ -7,11 +7,62 @@ export default Route.extend({
     this.container = options.container;
   },
 
-  render() {
-    this.view = new View({
-      model: new Model(),
-      editing: true
-    });
-    this.container.show(this.view);
-  }
+  render(params) {
+    var that = this;
+    function renderPage(_model){
+      that.view = new View({
+        model: _model,
+      });
+      that.container.show(that.view);
+    }
+    function retrieveListAndRender(_model){
+      var listxhr = new XMLHttpRequest();
+      listxhr.onreadystatechange = function(){
+        var list = JSON.parse(listxhr.responseText)["data"]
+        _model.attributes["organizationList"] = list;
```

Comment:
You should never explicitly assign attributes. Use `model.set(key, value)` instead.